modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -698,6 +698,11 @@ namespace detail
         static const char* tag() { return "gapi.use_only"; }
     };
 } // namespace detail
+
+#ifdef OPENCV_BINDINGS_PARSER
+GAPI_EXPORTS_W GCompileArgs compile_args(cv::gapi::GKernelPackage pkg);
+#endif // OPENCV_BINDINGS_PARSER
```

Comment:
Is it ok to have such conditionals in the public API headers? cc: @alalek

Reply:
@TolyaTalamanov , please move to something like `misc/python/shadow_gapi.hpp` (see https://github.com/opencv/opencv/blob/master/modules/python/bindings/CMakeLists.txt and https://github.com/opencv/opencv/blob/master/modules/core/misc/python/shadow_umat.hpp)

Reply:
Done